.pre-commit-config.yaml

```
@@ -11,14 +11,14 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v5.0.0
     hooks:
-      - id: trailing-whitespace
-      - id: end-of-file-fixer
-      - id: mixed-line-ending
-        args:
-          - --fix=lf
+      - id: check-added-large-files
       - id: check-toml
       - id: check-yaml
```

lets also check json files while we are at it

```suggestion
      - id: check-yaml
      - id: check-json
        exclude: '.+/tsconfig.json'
```